app/src/services/auth.services.ts

```diff
@@ -2,22 +2,27 @@ import axios from "axios";
 
 // Interfaces
 import { CCApp } from "../interfaces/CCApp";
+import { boolean } from "yup";
 
 export interface ContentCreatorApplication {
-  firstName: String;
-  lastName: String;
+  name: String;
   email: String;
-  motivation: String;
+  password: String;
```

Comment:
Might consider using another model for this. As the used model (contentCreatorApplication) is not the correct model for a content creator.

Reply:
I still see this